packages/haul-cli/src/commands/init.ts

```diff
@@ -1,5 +1,327 @@
-import { Runtime } from '@haul-bundler/core';
-import legacyInitCommand from '@haul-bundler/core-legacy/build/commands/init';
+import {
+  Runtime,
+  getReactNativeVersion,
+  DEFAULT_CONFIG_FILENAME,
+} from '@haul-bundler/core';
+import ora from 'ora';
+import fs from 'fs';
+import path from 'path';
+import inquirer from 'inquirer';
+import semver from 'semver';
+import dedent from 'dedent';
+
+const delay = (time: number) =>
+  new Promise(resolve => setTimeout(resolve, time));
+
+async function checkProject(cwd: string, runtime: Runtime) {
+  const progress = ora('Checking project files').start();
+  await delay(1000);
```

Comment:
why artificially wait?

Reply:
No idea, it was like this in original `init`. I assume they wanted to show that it's doing something, otherwise it just finished almost immediately.